### ISSUE TITLE
[android] Swiping the app does not restart main process

### ISSUE THREAD

**Issue by mrquincle** [crownstone/CrownstoneApp#40]
When swiping other apps, e.g. [Telegram](https://github.com/DrKLO/Telegram), the behaviour is as in the attached pictures.

What needs to be implemented is the ability to start the app from the service. There are examples on [stackoverflow](https://stackoverflow.com/questions/3456034/how-to-start-an-activity-from-a-service) that all describe `FLAG_ACTIVITY_NEW_TASK` and where Marcil Gil also nicely explains that this is a method that is very logical to have. It's intentional behaviour to have a service being able to launch something - even on the foreground. His example is a background service that listens to incoming calls. When a call is coming in an activity has to be started. The service itself can't pick up the phone.

TLDR: **Start the app from the service**
